Virtual Arsenal Manager

R3voA3 opened the pull request:
This pull request/branch will be merged once the Virtual Arsenal Manager is ready for the public.

To-do:
- [ ] Translation
- [ ] UI polishing. Moving to pixelGrid and making sure UI works on all resolutions/ui sizes
- [ ] Cleaning up functions

Commit:
- Increased 3den Radio UI height

Commit:
- Fixed Entity Counter positioning

Commit:
- Updated files

Commit:
- Updated File

Commit:
- Improved Code

Commit:
- Replaced GUI_GRID with pixelGrid

Commit:
- Replaced GUI_GRID with pixelGrid

Commit:
- Replaced GUI_GRID with pixelGrid

R3voA3 commented:
![20230926235340_1](https://github.com/R3voA3/3den-Enhanced/assets/17484252/ff77aac6-b37d-4aa0-900f-1401e5abaa4e)


Updated Presets UI.

Commit:
- Code formatting
- Added macros

Commit:
- Code formatting
- Added macros

Commit:
- Code formatting
- Added macros

Commit:
- Removed redundant macro

linkion commented:
I think we should remove the stats for weapons

Don't think it's really needed

linkion commented:
Maybe that space can be used to have the icon and compatible items to select

```
┌─────────────────────────────┬────────────────────────────┐
│ MAIN TV Tree                │ Selected Item Figure       │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             ├────────────────────────────┤
│                             │ Selected item Accessories  │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
│                             │                            │
└─────────────────────────────┴────────────────────────────┘
```

R3voA3 commented:
I have already started with a redesign.

Commit:
- Removed deprecated macro use

linkion commented:
Ok, but I think that they should be split horizontally. The tv tree doesn't take that much space width wise, but it is very vertical and requires a lot of scrolling when you expand something.

R3voA3 commented:
Final update for tonight. @linkion 

![20230928235416_1](https://github.com/R3voA3/3den-Enhanced/assets/17484252/7efda3e9-7ae9-4cb4-83fc-0b43c553823e)

Commit:
- Entire redesign of VAM main UI

Commit:
- Entire redesign of VAM main UI

Commit:
- Entire redesign of VAM main UI

Commit:
- Entire redesign of VAM main UI

Commit:
- Update

Commit:
- Update

Commit:
- Update

linkion commented:
Nvm, that looks good

Commit:
- added functionality to UI controls

Commit:
- added functionality to UI controls

Commit:
- added functionality to UI controls

Commit:
- added functionality to UI controls

Commit:
- Header and Close button in Placement Tools UI is now static

Commit:
- Updated UI layout

Commit:
- Updated UI layout

Commit:
- Updated UI layout

Commit:
- Updated UI layout

Commit:
- Updated code

Commit:
- Updated code

Commit:
- Updated code

Commit:
- Updated code

Commit:
- Added macro

Commit:
- Removed obsolete tools

Commit:
- Minor code improvements

Commit:
- Removed obsolete tools

Commit:
- Removed obsolete tools

Commit:
- Updated Files

Commit:
- Fixed incorrect class name of Placemen Tools UI

Commit:
- Remove multiselect from Variable Viewer
- Fixed list height
- Changed delete button icon

Commit:
- Profilenamespace is now saved if playlist was changed

Commit:
- Comment

Commit:
- Update

Commit:
- Update

Commit:
- Mission Backup attribute will now create backup files in mission root in a folder called ".enh_mission_sqm_backups"
- Existing backups will not be moved

Commit:
- Removed duplicated macros

Commit:
- Removed redundant definition

Commit:
- Texture Finder can no longer be dragged

Commit:
- Updated files

R3voA3 committed:
Merge pull request #284 from R3voA3/7.8.3

7.8.3

Commit:
- Bumped version number

Commit:
- Added missing define

Commit:
- Improved display3DENPublishMissionSelectImage

Commit:
- Removed wrong comment

Commit:
- Restructured context menu

Commit:
- Updated changelog

Commit:
- Readded Y axis color to statusbar

Commit:
- Updated files

Commit:
- fixed entity counter

Commit:
- updted files

R3voA3 committed:
Merge pull request #287 from R3voA3/v7.8.4

- Bumped version number

klausman committed:
Fix header formatting in CHANGELOG.md

The `#`/`##`/... leaders for headings must have a space after it to work correctly.

R3voA3 committed:
Merge pull request #292 from klausman/patch-1

Fix header formatting in CHANGELOG.md

hypoxia125 committed:
Update markerHideOnStart.hpp

Changed so that initial alpha can be saved and referenced

hypoxia125 committed:
Update markerHideOnStart.hpp

Commit:
- Added minimap

Commit:
- Added minimap

R3voA3 committed:
Merge pull request #293 from hypoxia125/master

[Patch] Allow markerHideOnStart to reference original alpha

Commit:
- improved code

Commit:
- improved code

Commit:
- added the ability to toggle minimap via menu strip

Commit:
- added the ability to toggle minimap via menu strip

Commit:
- fixed attributes would be saved to mission.sqm even if unused

Commit:
- fixed attributes would be saved to mission.sqm even if unused

Commit:
- updated files

Commit:
- updated files

Commit:
- fixed formatting

R3voA3 committed:
Merge branch 'master' into v7.8.6

Commit:
- restored changelog

Commit:
- renamed slider control so it's more generic

Commit:
- renamed slider control so it's more generic

Commit:
- Added the ability to change minimap size via shortcut
- Minimap is now also hidden if interface is invisible

Commit:
- Added the ability to change minimap size via shortcut
- Minimap is now also hidden if interface is invisible

Commit:
- Moved minimap to the left side to prevent overlapping with preview images

Commit:
- Moved minimap to the left side to prevent overlapping with preview images

Commit:
- added localiation for size settings
- changed default control for scale

Commit:
- added localiation for size settings
- changed default control for scale

Commit:
- code formatting

Commit:
- code formatting

Commit:
- replaced default player icon with custom one
- tweaked scaling

Commit:
- replaced default player icon with custom one
- tweaked scaling

Commit:
- update

Commit:
- update

klausman committed:
Mission backups: allow configuration of backup folder

This allows the user to choose which folder to copy the mission.sqm
backups to.

Addresses https://github.com/R3voA3/3den-Enhanced/issues/295

Commit:
- improved backup naming to include worldName
- changed control from EditShort to Edit
- added translations
- code formatting

Commit:
- updated changelog

Commit:
- updated contributors

R3voA3 committed:
Merge pull request #296 from klausman/backuppath

Mission backups: allow configuration of backup folder

Commit:
- improved local build handling to exclude signing

Commit:
- improved local build handling to exclude signing

Commit:
Merge branch 'v7.8.6' of https://github.com/R3voA3/3den-Enhanced into v7.8.6

Commit:
- code formatting

Commit:
- updated dynamic simulation debug mode

Commit:
- added tool to highlight area markers in 3D space

Commit:
- update

Commit:
- update

R3voA3 committed:
Merge pull request #294 from R3voA3/v7.8.6

v7.8.6

Commit:
- updated build task to support building without starting the game

Commit:
- minimap is disabled by default

Commit:
- update

R3voA3 committed:
Update README.md

Commit:
- updated documenation function

Commit:
- fixed bug in debug options function

Commit:
- added dev folder is being created on build with static name

Commit:
- added shortcut to Steam WS

Commit:
Merge branch 'master' of https://github.com/R3voA3/3den-Enhanced

Commit:
- Various improvements to Functions Viewer

Commit:
- Garrison UI was overhauled

Commit:
- Many tweaks to functions viewer

Commit:
- Fixed file name

Commit:
- fixed duplicated entry
- changed garrison ui name
- added icon

Commit:
- improved translation

Commit:
- fixed script error when deleting area helper
- garrison information is now updated every 10 frames

Commit:
- renamed function

Alfred-Neuman committed:
Translate French

Alfred-Neuman committed:
Translate French

R3voA3 committed:
Merge pull request #301 from Alfred-Neuman/master

Translate French

Commit:
- Restructured mod to work with hemtt

Commit:
- Remove some unused strings

Commit:
- updated .gitignore

Commit:
Merge branch 'Virtual-Arsenal-Manager' of https://github.com/R3voA3/3den-Enhanced into Virtual-Arsenal-Manager

Commit:
- moved functions

Commit:
- moved functions

Commit:
- moved functions

Commit:
- moved functions

Commit:
- deleted folder

Commit:
- deleted WIP folder as it's part of other branch now

Commit:
- removed old folder

Commit:
- removed old folder

Commit:
Merge branch 'Virtual-Arsenal-Manager' of https://github.com/R3voA3/3den-Enhanced into Virtual-Arsenal-Manager
